applications/nrf5340_audio/src/modules/audio_sync_timer_rtc.c

```diff
@@ -66,10 +70,7 @@ static uint32_t timestamp_from_rtc_and_timer_get(uint32_t ticks, uint32_t remain
 
 uint32_t audio_sync_timer_capture(void)
 {
-	nrf_rtc_task_trigger(audio_rtc_timer_instance.p_reg,
-				AUDIO_RTC_TIMER_CURR_TIME_CAPTURE_TASK);
-	nrf_timer_task_trigger(NRF_TIMER1,
-		nrf_timer_capture_task_get(AUDIO_SYNC_TIMER_CURR_TIME_CAPTURE_CHANNEL));
+	nrf_egu_task_trigger(NRF_EGU0, NRF_EGU_TASK_TRIGGER0);
 
 	uint32_t tick = nrf_rtc_cc_get(audio_rtc_timer_instance.p_reg,
```

Comment:
We need to look into how to handle the following statement in the product specification:
> There is a delay of 6 PCLK16M periods from when the [TASKS_CAPTURE[n]](https://infocenter.nordicsemi.com/topic/ps_nrf5340/rtc.html?cp=4_0_0_6_27_8#register.TASKS_CAPTURE) is triggered until the corresponding [CC[n]](https://infocenter.nordicsemi.com/topic/ps_nrf5340/rtc.html?cp=4_0_0_6_27_8#register.CC) register is updated.

Reply:
It means `first_read ` and `second_read` in the code below may have different values.  @henrla, can you confirm this?
```c
NRF_RTC0->TASKS_CAPTURE[0] = 1;
uint32_t first_read = NRF_RTC0->CC[0];
uint32_t second_read = NRF_RTC0->CC[0];
```

Reply:
Yes, this is my experience. On a branch where I did some drafting I did this to ensure the value was updated: https://github.com/nrfconnect/sdk-nrf/pull/13146/commits/1dc17ce81b43911f5451fd25b6d139e1721fea2b#diff-faf6a33c5ff79f996d75031737d189204a7743eb5013cd1f2f80aa53d0eee7d8R99

Reply:
updated